test/specs/parsing/functions/builtinFunctions.context.tests.js

```diff
@@ -0,0 +1,29 @@
+import slimdom from 'slimdom';
+
+import {
+	evaluateXPathToString
+} from 'fontoxpath';
+
+let documentNode;
+beforeEach(() => {
+	documentNode = slimdom.createDocument();
+});
+
+describe('Context related functions', () => {
+	describe('fn:current-dateTime', () => {
+		it('returns the current dateTime',
+			() => chai.assert.isOk(evaluateXPathToString('current-dateTime()', documentNode)));
```

Comment:
Could you check if the current-dateTime returns the same for an execution, but something else after a while?